apps/dashboard/src/main/java/com/akto/listener/InitializerListener.java

```diff
@@ -1968,5 +1972,23 @@ public void run() {
             }
         }, 0, 1, UsageUtils.USAGE_CRON_PERIOD);
     }
+
+    public void deleteTestRunsScheduler(){
+        scheduler.scheduleAtFixedRate(new Runnable() {
+            public void run(){
+                List<DeleteRunsInfo> deleteRunsInfos = DeleteRunsInfoDao.instance.findAll(Filters.empty());
+                if(deleteRunsInfos.size() > 0){
```

Comment:
> 0 check is redundant

Reply:
Removed it, but still checking for null .